docs/gc.md

```diff
@@ -83,7 +86,7 @@ The local GC scavenge reduces pause times by freqently reclaiming memory in the
 
 A special mode of the `gencon` policy is known as *Concurrent Scavenge*, which aims to minimize the time spent in stop-the-world pauses by collecting nursery garbage in parallel with running application threads. To enable Concurrent Scavenge, see [-Xgc:concurrentScavenge](xgc.md#concurrentscavenge).
```

Comment:
consider something like:

...which aims to further reduce average time spent...

'further' is in relative terms to just plain gencon with STW Scavenge, which already claimed that minimizes GC pause times.

Reply:
While talking about pauses, we could/should also (be fair to) mention it does _not_ improve the worst case pause time (it's about the same as plain Gencon).

Reply:
Update to:

"A special mode of the `gencon` policy is known as *Concurrent Scavenge*. This mode aims to further reduce the average time spent in stop-the-world pauses by collecting nursery garbage in parallel with running application threads. Whilst aiming to reduce the average time, this mode does not improve the worst case pause time when compared to running `gencon` without Concurrent Scavenge enabled."